sdk/core/System.ClientModel/src/Pipeline/ClientPipeline.cs

```diff
@@ -33,8 +37,12 @@ private ClientPipeline(ReadOnlyMemory<PipelinePolicy> policies, int perCallIndex
         _perCallIndex = perCallIndex;
         _perTryIndex = perTryIndex;
         _beforeTransportIndex = beforeTransportIndex;
+
```

Comment:
nit: extra line break

Reply:
I like this one stylistically, because the perXx indexes are a logical unit.